build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+set -x
+
+THISDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null 2>&1 && pwd )"
+
+VENVDIR=.venv
+
+if [ -z "${VIRTUAL_ENV}" ]; then
+    echo "Virtual environment has been not activated"
+    if ! [ -d "${THISDIR}/${VENVDIR}" ]; then
+        echo "Create new virtual environment"
+        python3.8 -m virtualenv -v --copies "${THISDIR}/${VENVDIR}"
+    fi
+fi
+        
+if [ -z "${VIRTUAL_ENV}" ]; then
+    . "${THISDIR}/${VENVDIR}/bin/activate"
+fi
+
+if ! pip list | grep PyYAML; then
+    pip install PyYAML
+fi    
+
+python download_data.py --data_dir data
```

Comment:
If you want user to install project using this script please mention it in the [readme](https://github.com/Samsung/CredData/#how-to-use)
If it's just for automation, than it can probably be a part of a CI job